src/firefly_exchange_client/client.py

```diff
@@ -789,6 +792,53 @@ async def get_user_leverage(self, symbol:MARKET_SYMBOLS, parentAddress:str=""):
         # todo fetch from exchange info route
         return 3
 
+    async def get_cancel_on_disconnect_timer(self,symbol:MARKET_SYMBOLS, parentAddress:str=""):
+        """
+            Returns a list of the user's countDowns for each market,
```

Comment:
for each market or for the provided market? the param `symbol` is a single market right?

Reply:
Yup it is for single market, fixing the comment